src/index.css

```diff
@@ -95,6 +95,18 @@ div:focus {
   filter: invert(0%);
 }
 
+@media screen and (min-width: 1800px) {
+.carousel-wrapper {
```

Comment:
I believe that this block and the one below it should be indented... This project is configured to use prettier if you are running VSCode you should be able to just download and run the extension for autoformatting of all files

Reply:
done!